collections/_posts/2022-11-14-DeformableConvolutionalNetworks.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: review
+title: "Deformable Convolutional Networks"
+tags:  Convolution, recepteive field, atrou
+author: "Nolann Lainé"
+cite:
+    authors: "Jifeng Dai, Haozhi Qi, Yuwen Xiong, Yi Li, Guodong Zhang, Han Hu, Yichen Wei"
+    title:   "Deformable Convolutional Networks"
+    venue:   "Microsoft Research Asia"
+pdf: "https://arxiv.org/pdf/1703.06211v3.pdf"
```

Comment:
I typicall prefer not to refer to a specific version of an arxiv paper (the `v3` at the end here) so that URL always leads to the most recent version.